### PR TITLE
vitables: init at 3.0.2

### PR DESCRIPTION
**Pull request by averyvigolo**
https://github.com/uvemas/ViTables/releases/tag/v3.0.2

###### Description of changes

Added the ViTables tool, which is a visualiser and editor for HDF5 files.

Most tests are disabled because they run GUIs.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by JulienMalka — CHANGES_REQUESTED** [review]
Hello, thanks for contributing ! Could you split your PR into 2 commits:
- maintainers: add williamvds
- vitables: init at 3.0.2

Conflicts also have to be fixed (rebase on master) before merging.

**Comment by williamvds** [comment]
> Hello, thanks for contributing ! Could you split your PR into 2 commits:
> 
>     * maintainers: add williamvds
> 
>     * vitables: init at 3.0.2
> 
> 
> Conflicts also have to be fixed (rebase on master) before merging.

Thanks, I've split the commit and rebased

**Comment by williamvds** [comment]
Updated to a more recent commit which supports Python 3.10, and gets rid of nose

File: pkgs/tools/misc/vitables/default.nix
Comment (puffnfresh) [inline pkgs/tools/misc/vitables/default.nix:12]:
```suggestion
    inherit version;
    owner = "uvemas";
    repo = "ViTables";
    rev = "08709d4dcf9266c8ed6d8669a27a18637c27d35e";
    sha256 = "sha256-oxMlFP8NHM4TGlLtpRxX9joPeW3VNv3CKzKBTdw2e7U=";
```

**Pull request closed without merging**
